builder/Dockerfile.centos-6

```diff
@@ -49,10 +52,6 @@ RUN yum -y update \
     zlib-devel \
     && yum clean all
 
-# Install pip
-RUN yum install -y epel-release && \
-   yum install -y python-pip
```

Comment:
I had to move this up since pcre2-devel comes from EPEL. Also, no changes to the CentOS 6 packaging script since it already had PCRE2 as a dependency (probably copied from CentOS 7)